game_eggs/steamcmd_servers/the_isle/egg-the-isle.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2022-01-05T15:57:23+01:00",
+    "name": "The Isle",
+    "author": "fredrik.johansenfuun@gmail.com",
+    "description": "The Isle is an open beta with an open-world survival game where players choose from three factions in an attempt to survive a fierce island. Hunt. Prey. Survive.",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/games:source"
+    ],
+    "file_denylist": [],
+    "startup": "\/home\/container\/run.sh",
```

Comment:
You can add the startup and no need to use a script

Reply:
I tried to do that but i couldn't get it to work properly so i went with what i know. 
Especially since there are so many parameters i just couldn't get all of them working in the startup. If you have any advice on how i could do it i'll happily change it.